sonaranalyzer-dotnet/.sonarlint/SonarAnalyzer.slconfig

```diff
@@ -9,7 +9,7 @@
   "Profiles": {
     "CSharp": {
       "ProfileKey": "AVxYKDdrFTbgxqUNcakm",
-      "ProfileTimestamp": "2019-05-07T17:01:40+02:00"
+      "ProfileTimestamp": "2019-10-29T09:27:27+01:00"
```

Comment:
Changes in this file are unrelated to the current PR. If this was intended, I think it deserves a separate PR